fenix/app/src/main/res/drawable/ic_bookmarks_search.xml

```diff
@@ -7,15 +7,8 @@
     android:height="24dp"
     android:viewportWidth="24"
     android:viewportHeight="24">
-    <path
-        android:name="path"
-        android:fillColor="@color/photonWhite"
-        android:pathData="M 2 0 L 22 0 C 22.53 0 23.039 0.211 23.414 0.586 C 23.789 0.961 24 1.47 24 2 L 24 22 C 24 22.53 23.789 23.039 23.414 23.414 C 23.039 23.789 22.53 24 22 24 L 2 24 C 1.47 24 0.961 23.789 0.586 23.414 C 0.211 23.039 0 22.53 0 22 L 0 2 C 0 1.47 0.211 0.961 0.586 0.586 C 0.961 0.211 1.47 0 2 0"
-        android:strokeWidth="1" />
-    <path
-        android:pathData="m7.225,11.324 l-2.86,-2.375a1.207,1.207 0,0 1,-0.378 -1.309,1.205 1.205,0 0,1 1.074,-0.835l3.697,-0.239 0.501,-0.375 1.364,-3.427A1.209,1.209 0,0 1,11.75 2c0.499,0 0.941,0.3 1.127,0.763v0.001l1.364,3.427 0.501,0.375 3.697,0.239c0.498,0.032 0.92,0.36 1.074,0.835 0.155,0.476 0.006,0.99 -0.378,1.309L16.28,11.32l-0.18,0.592 0.91,3.575c0.123,0.485 -0.059,0.986 -0.463,1.279a1.202,1.202 0,0 1,-1.359 0.046l-3.127,-1.977h-0.621l-3.126,1.977c-0.426,0.266 -0.96,0.247 -1.361,-0.045a1.205,1.205 0,0 1,-0.463 -1.279l0.908,-3.566 -0.173,-0.598z"
-        android:fillColor="@color/photonBlack"/>
-    <path
-        android:pathData="m5.782,20.5 l-0.8,-0.8v-1.95a0.75,0.75 0,0 0,-1.5 0v1.75a2.5,2.5 0,0 0,2.5 2.5h11.535a2.5,2.5 0,0 0,2.5 -2.5v-1.75a0.75,0.75 0,0 0,-1.5 0v1.95l-0.8,0.8H5.782z"
-        android:fillColor="@color/photonBlack"/>
+  <path
```

Comment:
Before you land this, could you add a reference to where these new icons are sourced from? This is mainly to provide some context for anybody in the future that is looking back at these icon changes and being able to determine the requirements and the source of these icons at the time for whatever reason they needed to do this. 

I am asking for this because there were no clear directions in the bug description about where to grab these icons from. Ideally, we would have this in the bug description, but I also see some benefits to linking a figma in the PR description as well.

Reply:
design (and icons) could be found [here](https://www.figma.com/file/epu0Sj6mCPk7K282sNpHgI/Site?node-id=874%3A14793&t=KJyX7Whx81P5rg5Z-0)